src/expr/core/src/error.rs

```diff
@@ -135,3 +141,38 @@ impl From<PbFieldNotFound> for ExprError {
         ))
     }
 }
+
+/// A collection of multiple errors.
+#[derive(Error, Debug)]
+pub struct MultiExprError(Box<[ExprError]>);
+
+impl MultiExprError {
+    /// Returns the first error.
+    pub fn first(self) -> ExprError {
+        self.0.into_vec().into_iter().next().expect("first error")
```

Comment:
```suggestion
        self.0.into_iter().next().expect("first error")
```

Reply:
If removed, the item would be `&ExprError`. Seems to be a historical problem https://github.com/rust-lang/rust/issues/59878